src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -380,7 +380,7 @@
 )
 
 ol_apps_profile_client_scope = keycloak.openid.ClientScope(
-    "profile", realm_id=ol_apps_realm.id
+    "ol-profile", realm_id=ol_apps_realm.id
 )
```

Comment:
It would be good to add a `name="ol-profile"` so that it does not add the random suffix